epochStart/bootstrap/interface.go

```diff
@@ -60,3 +59,16 @@ type NodeTypeProviderHandler interface {
 	GetType() core.NodeType
 	IsInterfaceNil() bool
 }
+
+// EpochStartBootstrapperFactoryHandler defines the epoch start bootstrapper factory handler
+type EpochStartBootstrapperFactoryHandler interface {
+	CreateEpochStartBootstrapper(epochStartBootstrapArgs ArgsEpochStartBootstrap) (EpochStartBootstrapper, error)
+	IsInterfaceNil() bool
+}
+
+// EpochStartBootstrapper defines the epoch start bootstrap functionality
+type EpochStartBootstrapper interface {
+	Bootstrap() (Parameters, error)
+	IsInterfaceNil() bool
```

Comment:
Switch the order between Bootstrap and IsInterfaceNil ?

Reply:
moved IsInterfaceNil to last position